editor/src/core/shared/template-path.ts

```diff
@@ -441,23 +441,32 @@ export function parentPath(path: InstancePath): TemplatePath
 export function parentPath(path: TemplatePath): TemplatePath | null
 export function parentPath(path: TemplatePath): TemplatePath | null {
   if (isScenePath(path)) {
-    return null
+    return null // FIXME This is wrong!
   } else {
     return instancePathParent(path)
   }
 }
 
 function elementPathToUID(path: ElementPath): id {
-  return last(path)!
+  return last(path) ?? ''
```

Comment:
hmmm. I don't like the `''` here because it can lead to more silent errors. at least a surprise `undefined` throws a runtime error, helping us find the problem. but a `'' === ''` comparison can survive in the codebase for years

Reply:
I _think_ this will still be a problem whilst we have the empty path used in some places for the storyboard root. I'll revert this and check nothing breaks, then make a note to remove all use of the empty path for the storyboard root.

Reply:
or maybe you could throw an error here if the path is empty?

Reply:
:+1: